### ISSUE TITLE
Enable use of arrow keys to select from the article list

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js-windows#55]
For keyboard users, as opposed to touch or mouse, selection of articles from the article list should be possible with the down and up arrow keys and Enter.

**Comment by mossroy** [comment]
I had the same idea upstream : kiwix/kiwix-js#55 , but did not work on it at all.

**Comment by Jaifroid** [comment]
Oh, cool, good to know. When I get time to look at it, I'll try to do it in a universal way that will work for both implementations (it's the same code base there).

**Comment by Jaifroid** [comment]
This issue has now been addressed upstream in PR https://github.com/kiwix/kiwix-js/pull/501.
